pkg/model/estimate.go

```diff
@@ -0,0 +1,99 @@
+package model
+
+import (
+    "os/exec"
+	"net"
+	"encoding/json"
+	"log"
+	// "os"
+)
+
+const (
+	SERVE_SOCKET = "/tmp/estimator.sock"
+)
+
+type Estimator struct {
+	Err chan error	
+	*exec.Cmd
+}
+
+type PowerRequest struct {
+	ModelName string   `json:"model_name"`
+	XCols 	[]string 	`json:"x_cols"`
+	XValues [][]float32 `json:"x_values"`
```

Comment:
What is XCols and XValues?
Might be nice the have more meaningful names or some comments here.

Reply:
You have it in the PR description, and might be good to have it in the code as well

Reply:
Please see example above.
About naming, how's about
XCols -->  MetricNames
XValues --> MetricValuesOfAllPods

Reply:
sgtm